ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -36,22 +39,54 @@ export const NewQuestionForm = (props: NewQuestionFormProps) => {
             />
           </div>
           <label className="form-label" htmlFor="questionType">Question type</label>
-          <select id="questionType" className="form-select" value={selectedQuestionType}
+          <select id="questionType" defaultValue="placeholder" className="form-select" value={selectedQuestionType}
```

Comment:
It's a little unusual to mix `defaultValue` (usually used for uncontrolled components) and `value` (which makes a component controlled).
https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components

I think I would suggest removing `defaultValue` here and using a string state variable (initialized to `placeholder`) for the value of this select menu.

As is, it looks like selecting `freetext` here gets into a weird state where `selectedQuestionType` is no longer valid until something is entered in the free text field.